### PR TITLE
Begin basic node class for s_db_admin

### PR DESCRIPTION
**Pull request by surminus**
We're looking to provision our databases in RDS rather than provision EC2 machines with databases on them.

To create, administer and backup databases using Puppet, we need something to use that can do this in our current Puppet estate.

For now just install packages to used to work with our current versions MySQL and PostgreSQL.

### CODE REVIEW

File: hieradata/common.yaml [hunk 1]
```diff
@@ -1311,6 +1311,8 @@ hosts::production::licensify::hosts:
     ip: '10.5.0.18'
 
 hosts::production::management::hosts:
+  db-admin-1:
```
Comment (SamLR) [inline hieradata/common.yaml]: Shouldn't this be in `hierdata_aws/common.yaml`?

Comment (surminus) [inline hieradata/common.yaml]: Probably. I only added this to get tests to pass, and actually I think I'd rather add an exception in the tests.

Comment (SamLR) [inline hieradata/common.yaml]: fair enough (we should probably look at rolling those tests out to cover stuff in `hieradata_aws` too but that's for another PR...)

Comment (surminus) [inline hieradata/common.yaml]: I think you're right.

Comment (SamLR) [inline hieradata/common.yaml]: otherwise LGTM : )

**Pull request merged**